server.js

```diff
@@ -12,6 +12,10 @@ app.get('/ping', (req, res) => {
   res.send('pong');
 });
 
+app.get('/posts/:id', (req, res) => {
+  res.json([{ 'id': req.params.id }]);
```

Comment:
- `posts/:id` should return single post not an array of posts
- why do you put quotes around `id`?